### ISSUE TITLE
font size and color props are not converted from polymorphic variants to strings

### ISSUE THREAD

**Issue by leeor** [mscharley/bs-material-ui-icons#17]
Unless I'm missing something, the simple bindings for each of the icons define font size and color to have the types defined in `Icon.re`, which are polymorphic variants with a `[@bs.deriving jsConverter]`.

However, there are not "translation" functions that actually call `fontSizeToJs` or `colorToJs`, making these two props unusable.

Did I miss something in how these bindings are supposed to be used? Or do the bindings need to be changed to something like (e.g., for fontSize):
```reason
  [@bs.module "@material-ui/icons/AccountBalanceOutlined"][@react.component]
  external make: (
    ~className: string=?,
    ~color: color=?,
    ~fontSize: [@bs.string][
       | [@bs.as "default"] `Default
       | [@bs.as "inherit"] `Inherit
       | [@bs.as "small"] `Small
       | [@bs.as "large"] `Large
      ]=?,
    ~nativeColor: string=?,
    ~style: ReactDOMRe.Style.t=?,
    ~titleAccess: string=?,
    ~viewBox: string=?,
  ) => React.element = "default";

```

Edit: I've submitted PR #18 with the fix suggested above.

**Comment by mscharley** [comment]
Hmm, in retrospect it seems I haven't tried to use this myself yet, the closest I could find is the following:

```
<IconButton color=`Inherit onClick={_ => OpenModal |> send}>
  <Icons.Help />
</IconButton>
```

I did copy this implementation from bs-material though so I would have expected it to work at one point. I'm not sure of the current state though as I haven't used this in some time. I would *expect* it to work as is though. Did you try to use it and have it fail to work? I don't have an active project at the moment using this package.

**Comment by leeor** [comment]
Yes, I tried using fontSize and got an error from material UI that some random number is not a valid value expected for fontSize.

The changes I suggested in the PR I submitted solved the issue.

AFAIK, defining a polymorphic variant type with `jsConverter` does not automatically call the conversion code when a value of that type is passed to an external. The only way I know of to do that is what I did in my PR.
